prospector/commit_processor/feature_extractor.py

```diff
@@ -141,6 +143,10 @@ def extract_contains_jira_reference(jira_references: "list[str]") -> bool:
 
 
 def extract_referred_to_by_nvd(commit: Commit, advisory_record: AdvisoryRecord) -> bool:
-    return (
-        advisory_record.from_nvd and commit.commit_id[:7] in advisory_record.description
+    response = requests.get(
```

Comment:
I guess in this cases it would make sense to pass the endpoint URL as a parameter to the function, so that we can inject the value.